README.md

```diff
@@ -42,6 +42,7 @@ This functionality is in beta and is subject to change. The design and code is l
 | `ioThreadCount`                 | Set the Zeebe Cluster IO thread count                                                                                                                                | `2`  
 | `zeebeCfg`                 | Can be used to set several zeebe configuration options.                                                                                                                                | `null`
 | `gatewayMetrics`                 | Enables the exporting of the gateway prometheus metrics                                                                                                                                | `false`
+| `brokerEmbeddedGatewayEnable`                 | Enables the embedded broker gateway                                                                                                                                | `false`
```

Comment:
```suggestion
| `embeddedGatewayEnabled`                 | Enables the embedded gateway in all brokers                                                                                                                                | `false`
```

Reply:
+1 @ezolnbl can you please add @npepinpe suggestions?